craco.config.js

```diff
@@ -92,6 +92,7 @@ module.exports = {
           http: require.resolve('stream-http'),
           https: require.resolve('https-browserify'),
           crypto: require.resolve('crypto-browserify'),
+          zlib: false,
```

Comment:
`@trezor/connect-plugin-ethereum` requires `@metamask/eth-sig-util` that requires `zlib`